<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QWidget" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>991</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="templateText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>991</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Enter text...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="templateTitle">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter title...</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>438</width>
        <height>38</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="showTitleListButton">
      <property name="font">
       <font>
        <family>Symbol</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>三</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QListWidget" name="titleList">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>0</y>
     <width>171</width>
     <height>351</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>新年の挨拶</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="Dense7Pattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>お礼文</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>依頼文</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ather1</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ather2</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ather3</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="closeTitleListButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
